src/prefect/settings.py

```diff
@@ -1321,6 +1321,17 @@ def default_cloud_ui_url(settings, value):
 Whether or not to enable experimental client side task run orchestration.
 """
 
+PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_CONCURRENCY = Setting(bool, default=True)
```

Comment:
maybe it helps with testing, but given that client-side orchestration will depend on client-side concurrency you might be able to get away with re-using that flag.

Reply:
Yeah, agreed. I've consolidated behind that flag now.